Rewording Cluster Autoscaling paragraph in the last blog (#73)
eclipse-che/blog: _posts/2024-11-26-in-kubernetes-we-trust.adoc

  controller.devfile.io/storage-type: ephemeral
....

=== Cluster Autoscaling

> Although link:https://kubernetes.io/docs/concepts/cluster-administration/cluster-autoscaling/[Cluster Autoscaling] is a powerful Kubernetes feature, you cannot always fall back on it, and should always consider predictive scaling by analyzing the load data on your environment to detect daily or weekly usage patterns. If your workloads follow some pattern and there are dramatic peaks throughout the day you should consider provisioning worker nodes accordingly (e.g. a lot of users turn on their smart speakers in the morning between 7 - 9 am, and there is a huge spike in the requests that on the infrastructure level is predicted and handled in advance).

Reviewer: ```suggestion
Although link:https://kubernetes.io/docs/concepts/cluster-administration/cluster-autoscaling/[Cluster Autoscaling] is a powerful Kubernetes feature, you cannot always fall back on it and should always consider predictive scaling by analyzing load data on your environment to detect daily or weekly usage patterns. If your workloads follow a pattern and there are dramatic peaks throughout the day you should consider provisioning worker nodes accordingly (e.g. a lot of users turn on their smart speakers in the morning between 7 - 9 a.m. causing a huge spike in requests that can be predicted on the infrastructure level and handled in advance).
```
